ansible-ipi-install/roles/node-prep/vars/main.yml

```diff
@@ -16,3 +16,5 @@ package_list:
   - policycoreutils-python3
   - https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-crypto-2.6.1-18.el8ost.x86_64.rpm
   - https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-pyghmi-1.0.22-2.el8ost.noarch.rpm
+  - httpd
+  - gzip
```

Comment:
because we doing the container, do we need these packages anymore? (httpd and gzip)

Reply:
Ah, good catch.  Nope!